ibm_was/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "display_name": "IBM WAS",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "ibm_was",
+  "metric_prefix": "ibmwas.",
+  "metric_to_check": "",
```

Comment:
You should add this

Reply:
This field should be optional. The check is coded where you could enable any of the "types" of metrics, so its never guaranteed that any individual metric will be reporting. For ex, one user may only have the basic JVM Runtime metrics coming in, while another may only have the Servlet Session metrics coming, etc.

Reply:
Are there any metrics that will be sent no matter what?

Reply:
It is possible to configure it to send nothing, yes

Reply:
@ofek @gmmeyer Yes, its possibly (while the default makes this unlikely) that the check could be configured in a way where no metrics are sent (but the service check would be sent no matter what)

Reply:
The metric to check will enable the tile by default when they begin sending data. I think it's optimal to fill it in if we can.

Reply:
I added one in.